seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/sqlite/SqliteTypeMapper.java

```diff
@@ -0,0 +1,176 @@
+    private static final String SQLITE_DECIMAL = "DECIMAL";
+    private static final String SQLITE_DECIMAL_UNSIGNED = "DECIMAL UNSIGNED";
+    private static final String SQLITE_FLOAT = "FLOAT";
+    private static final String SQLITE_FLOAT_UNSIGNED = "FLOAT UNSIGNED";
+    private static final String SQLITE_DOUBLE = "DOUBLE";
+    private static final String SQLITE_DOUBLE_PRECISION = "DOUBLE PRECISION";
+    private static final String SQLITE_DOUBLE_UNSIGNED = "DOUBLE UNSIGNED";
+    private static final String SQLITE_NUMERIC = "NUMERIC";
+    private static final String SQLITE_REAL = "REAL";
+
+    // -------------------------text----------------------------
+    private static final String SQLITE_CHAR = "CHAR";
+    private static final String SQLITE_CHARACTER = "CHARACTER";
+    private static final String SQLITE_VARYING_CHARACTER = "VARYING_CHARACTER";
+    private static final String SQLITE_NATIVE_CHARACTER = "NATIVE_CHARACTER";
+    private static final String SQLITE_NCHAR = "NCHAR";
+    private static final String SQLITE_VARCHAR = "VARCHAR";
+    private static final String SQLITE_LONGVARCHAR = "LONGVARCHAR";
+    private static final String SQLITE_LONGNVARCHAR = "LONGNVARCHAR";
+    private static final String SQLITE_NVARCHAR = "NVARCHAR";
+    private static final String SQLITE_TINYTEXT = "TINYTEXT";
+    private static final String SQLITE_MEDIUMTEXT = "MEDIUMTEXT";
+    private static final String SQLITE_TEXT = "TEXT";
+    private static final String SQLITE_LONGTEXT = "LONGTEXT";
+    private static final String SQLITE_JSON = "JSON";
+    private static final String SQLITE_CLOB = "CLOB";
+
+    // ------------------------------time(text)-------------------------
+    private static final String SQLITE_DATE = "DATE";
+    private static final String SQLITE_DATETIME = "DATETIME";
+    private static final String SQLITE_TIME = "TIME";
+    private static final String SQLITE_TIMESTAMP = "TIMESTAMP";
+
+    // ------------------------------blob-------------------------
+    private static final String SQLITE_TINYBLOB = "TINYBLOB";
+    private static final String SQLITE_MEDIUMBLOB = "MEDIUMBLOB";
+    private static final String SQLITE_BLOB = "BLOB";
+    private static final String SQLITE_LONGBLOB = "LONGBLOB";
+    private static final String SQLITE_BINARY = "BINARY";
+    private static final String SQLITE_VARBINARY = "VARBINARY";
+    private static final String SQLITE_LONGVARBINARY = "LONGVARBINARY";
+
+    @Override
+    public SeaTunnelDataType<?> mapping(ResultSetMetaData metadata, int colIndex) throws SQLException {
+        String columnTypeName = metadata.getColumnTypeName(colIndex).toUpperCase().trim();
+        switch (columnTypeName) {
+            case SQLITE_BIT:
+            case SQLITE_BOOLEAN:
+                return BasicType.BOOLEAN_TYPE;
+            case SQLITE_TINYINT:
+            case SQLITE_TINYINT_UNSIGNED:
+            case SQLITE_SMALLINT:
```

Comment:
We also support short type, you can use it to map smallint and tinyint.

Reply:
thx, I'll test it.